### DISCUSSION TITLE
Pytorch v2 compatibility

### DISCUSSION THREAD

**Discussion by AntonOfTheWoods** [mrdbourke/pytorch-deep-learning#389] [General]
There are a couple of entries in the log about pytorch2 but nothing suggesting we should expect it to work or not with the additions made. Am I going to get 4 hours in and nothing is going to work? v2 is now the default in colab.

**Comment by mrdbourke** [comment]
Hey @AntonOfTheWoods,

PyTorch 2.0 is an additive release (new features) and is backwards compatible (works with older versions too).

This means all previous code should still work.

I'm about to finish the tutorial on PyTorch 2.0, see it here: https://www.learnpytorch.io/pytorch_2_intro/

The main update with PyTorch 2.0 is via `torch.compile()`, it provides potential speedups to your models behind the scenes with newer GPU hardware.

**Reply by AntonOfTheWoods** [reply]
I'm currently about hour 14 and have only noticed one thing so far (though I decided to go the nvidia-docker+jupyyer route rather than Colab), and that was with `Accuracy` (needs a type now, no default) which isn't related to the 2.0 anyway.

Thanks for an awesome course!
